shoop/core/models/products.py

```diff
@@ -252,12 +252,40 @@ def get_shop_instance(self, shop):
 
         return shop_inst
 
-    def get_cheapest_child_price(self, context, quantity=1):
+    def get_child_price_range(self, context, quantity=1):
+        """
+        Get the prices for cheapest and the most expensive child
+
+        The attribute used for sorting is `PriceInfo.price`.
+
+        :type context: shoop.core.contexts.PriceTaxContext
+        :type quantity: int
+        :return: a tuple of prices
+        :rtype: (shoop.core.pricing.Price, shoop.core.pricing.Price)
+        """
+        infos = sorted(
+            [c.get_price_info(context, quantity=quantity) for c in self.variation_children.all()],
+            key=lambda x: x.price
+        )
+        return infos[0].price, infos[len(infos)-1].price
+
+    def get_cheapest_child_price_info(self, context, quantity=1):
+        """
+        Get the `PriceInfo` of the cheapest variation child
+
+        The attribute used for sorting is `PriceInfo.price`.
+
+        :type context: shoop.core.contexts.PriceTaxContext
+        :rtype: shoop.core.pricing.PriceInfo
+        """
         return sorted(
-            c.get_price(context, quantity=quantity)
-            for c in self.variation_children.all()
+            [c.get_price_info(context, quantity=quantity) for c in self.variation_children.all()],
+            key=lambda x: x.price
         )[0]
 
+    def get_cheapest_child_price(self, context, quantity=1):
```

Comment:
Docstring? Also, you could move this function back where it used to be (now replaced by `get_child_price_range`)